pkgs/development/python-modules/wxPython/4.1.nix

```diff
@@ -51,24 +81,17 @@ buildPythonPackage rec {
         ("pangocairo", "${pango.out}/lib/libpangocairo-1.0.so"),
```

Comment:
The above substituteInPlace can be combined.

Reply:
done